.test/config.yaml

```diff
@@ -1,6 +1,8 @@
 samples: samples.tsv
 units: units.tsv
 
+# trimming optional: yes or no
+trim: yes
```

Comment:
I suggest:

```suggestion
adapter:
  trim: true
  sequence: ACGGATCGATCGATCGATCGAT
```

Reply:
On the other hand, trimming might also involve non adapter sequences, right? E.g. quality trimming, depending on how people configure cutadapt. So maybe rather:

```suggestion
trimming:
  skip: false
  adapter: ACGGATCGATCGATCGATCGAT
```

Reply:
ok new merge request now